packages/block-library/src/file/edit.js

```diff
@@ -21,6 +21,7 @@ import {
 	MediaPlaceholder,
 	BlockControls,
 	RichText,
+    PlainText,
```

Comment:
There is mixed whitespace here which is failing the build. Can you correct this to use tabs instead of spaces?